support/doc/development/tests.md

```diff
@@ -42,7 +48,7 @@ $ npm run test # See scripts/test.sh to run a particular suite
 Most of tests can be runned using:
 
 ```bash
-TS_NODE_TRANSPILE_ONLY=true mocha -- --timeout 30000 --exit -r ts-node/register -r tsconfig-paths/register --bail server/tests/api/videos/video-transcoder.ts
+TS_NODE_TRANSPILE_ONLY=true npx mocha -- --timeout 30000 --exit -r ts-node/register -r tsconfig-paths/register --bail server/tests/api/videos/video-transcoder.ts
```

Comment:
prefer `npm run mocha -- timeout...`

Reply:
Maybe I am wrong, but if you use «mocha», it will use the version installed globally. That is not necessarely compatible with the version used by Peertube.
Moreover, if I have no version installed globally, I got an error «bash: /usr/local/bin/mocha: Aucun fichier ou dossier de ce type»

Reply:
`npm run mocha` will run `mocha` script defined in `package.json`

Reply:
Oops, I misread your answer. Ok.